src/wam2layers/preprocessing/pressure_levels.py

```diff
@@ -1,13 +1,12 @@
-from wam2layers.preprocessing.utils import interpolate, sortby_ndarray
-
-
 import numpy as np
 import xarray as xr
 
+from wam2layers.preprocessing.utils import interpolate, sortby_ndarray
+
 
 def extend_pressurelevels(
-    input: xr.Dataset,
-    surface: xr.Dataset,
+    level_data: xr.Dataset,
```

Comment:
Ahh, I commented on this in the previous commit, but here you already address it :-)